packages/f/formeo.json

```
@@ -0,0 +1,32 @@
+{
+  "name": "formeo",
+  "filename": "formeo.min.js",
+  "description": "A zero dependency JavaScript module for drag and drop form creation.",
+  "keywords": [
+    "drag and drop",
+    "form builder",
+    "form maker",
+    "forms",
+    "javascript"
+  ],
+  "homepage": "https://formeo.io",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/Draggable/formeo"
+  },
+  "license": "MIT",
+  "author": "Draggable https://draggable.io",
```

```suggestion
  "authors": [
    {
      "name": "Draggable",
      "url": "https://draggable.io"
    }
  ],
```
We take a slightly different structure for the authors